pipeline-runner/R/gem2s-2-load_cellranger_files.R

```diff
@@ -101,6 +111,34 @@ format_annot <- function(annot_list) {
 }
 
 
+#' Calls BD rhapsody data parsing functions
+#'
+#' Currently we only have support for sparse expression matrices.
+#'
+#' @inheritParams download_user_files
+#'
+#' @return list containing \itemize{
+#'   \item{"counts_list"}{named list of dgCMatrix per sample}
+#'   \item{"annot"}{data.frame with gene symbols}
+#'
+call_read_rhapsody <- function(config, input_dir) {
```

Comment:
why having `call_` in the names of these? You can call them `read_10x_files` and `read_rhapsody_matrix_files`